.github/workflows/release-nix.yml

```
@@ -0,0 +1,58 @@
+name: "Release nix"
```

```suggestion
name: "Master Push"
```

Also rename the file to `master-push.yml`. We'll be adding more jobs to it soon.